plugins/woocommerce/src/Checkout/Helpers/ReserveStock.php

```diff
@@ -202,18 +213,28 @@ private function reserve_stock_for_product( $product_id, $stock_quantity, $order
 	 */
 	private function get_query_for_reserved_stock( $product_id, $exclude_order_id = 0 ) {
 		global $wpdb;
+
+		$join         = "$wpdb->posts posts ON stock_table.`order_id` = posts.ID";
+		$where_status = "posts.post_status IN ( 'wc-checkout-draft', 'wc-pending' )";
+		if ( $this->is_cot_in_use() ) {
```

Comment:
Not essential you change this, but if you were to import `Automattic\WooCommerce\Utilities\OrderUtil` you could do:

```suggestion
		if ( OrderUtil::custom_orders_table_usage_is_enabled() ) {
```

(and then delete the `is_cot_in_use()` method.)

Reply:
I like that better, thanks! 5d59e534dab35d24ac01205fb633c07bcf07aadc